docs/en/query_language/agg_functions/combinators.md

```diff
@@ -44,5 +44,71 @@ Merges the intermediate aggregation states in the same way as the -Merge combina
 
 Converts an aggregate function for tables into an aggregate function for arrays that aggregates the corresponding array items and returns an array of results. For example, `sumForEach` for the arrays `[1, 2]`, `[3, 4, 5]`and`[6, 7]`returns the result `[10, 13, 5]` after adding together the corresponding array items.
 
+## -Resample
+
+Allows to aggregate data, separating them into intervals.
```

Comment:
this sentence sounds confusing and hardly makes it possible to understand the feature without reading further, let's make it more detailed?